wrphttp/handler.go

```diff
@@ -128,6 +129,15 @@ func (wh *wrpHandler) ServeHTTP(httpResponse http.ResponseWriter, httpRequest *h
 		return
 	}
 
+	if entity.Message.Type == 3 && entity.Message.TransactionUUID == "" {
```

Comment:
Aren't there additional message types that require a `TransactionUUID`?  I thought all the CRUD ones do as well.

In any case, I would use the `MessageType.SupportsTransaction` method to determine if it has a message type:

```
if entity.Message.Type.SupportsTransaction() && len(entity.Message.TransactionUUID) == 0 {
    // etc
}
```

Given that we're requiring a transaction UUID now, that method should probably change to something clearer, like `RequiresTransactionUUID`.